### PR TITLE
initial AppVeyor support

### PR DESCRIPTION
**Pull request by chipitsine**
there're still unresolved things like

a) what to do with missing "man2html"
b) what to do with missing makensis 

but, I beleive, it is already useful

### CODE REVIEW

**Comment by chipitsine** [comment]
Up

**Comment by mattock** [comment]
I'd just remove the commented build-complete/build-snapshot/man2html lines from this PR. They can be added back in if/when we can actually use them.

Do you have a link to AppVeyor output produced by this commit? I'd like to see this in action.

**Comment by chipitsine** [comment]
there are caveats in mingw world, @selvanair mentioned them at https://github.com/OpenVPN/openvpn-gui/pull/151

I think running mingw builds regularly is rather good idea, yet we need to discuss how to deal with those caveats

**Comment by chipitsine** [comment]
I cannot remember why I commented out ""%CYGWIN%\\bin\\bash -lc 'cp `which true` /usr/bin/man2html'"", I will uncomment it and rebuild

as for the rest, ok, we can remove comments as well

**Comment by chipitsine** [comment]
https://ci.appveyor.com/project/chipitsine/openvpn-build/build/cygwin-31

**Comment by chipitsine** [comment]
up

File: .appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - ARCH: x86
+      CYGWIN: C:\Cygwin
+      CHOST: i686-w64-mingw32
+    - ARCH: x86_64
+      CYGWIN: C:\Cygwin64
+      CHOST: x86_64-w64-mingw32
+
+build_script:
+  - "%CYGWIN%\\setup-%ARCH%.exe -g -q -P mingw64-i686-openssl,mingw64-x86_64-openssl,cygwin-devel,unzip,dos2unix"
+  - "%CYGWIN%\\bin\\bash -lc 'cp `which true` /usr/bin/man2html'"
```
Comment (chipitsine) [inline .appveyor.yml]: I haven't found an elegant solution. 
I'd leave it for the community :) someone will come with good idea, I guess

**Pull request merged**